root/static/scripts/edit/URLCleanup.js

```diff
@@ -1540,6 +1540,13 @@ const CLEANUPS = {
       return /^https:\/\/musopen\.org\/music\/\d+\/$/.test(url);
     },
   },
+  'muziekweb': {
+    match: [new RegExp('^(https?://)?www\\.muziekweb\\.eu/Link/', 'i')]
```

Comment:
Missing trailing comma (which breaks parsing, so it's a proper bug!)

Reply:
This should be fixed now.